public/icons/shop-placeholder.svg

```diff
@@ -0,0 +1,19 @@
+<path d="M233.298 95.3881C232.26 94.6622 231.107 94.2993 229.897 94.2993C229.493 94.2993 229.089 94.3598 228.628 94.4203C227.014 94.7832 225.63 95.751 224.708 97.2027L214.791 112.929C213.869 114.381 213.581 116.075 213.869 117.768C214.215 119.462 215.137 120.914 216.521 121.881C217.905 122.849 219.519 123.152 221.133 122.849C222.748 122.486 224.131 121.519 225.054 120.067L234.97 104.34C235.893 102.889 236.181 101.195 235.893 99.5012C235.604 97.8076 234.682 96.3559 233.298 95.3881Z" fill="white"/>
+<path d="M165.037 104.34L174.953 120.067C175.876 121.519 177.259 122.486 178.874 122.849C180.488 123.212 182.102 122.849 183.486 121.882C186.311 119.885 187.118 115.893 185.216 112.929L175.299 97.2027C174.377 95.751 172.993 94.7832 171.379 94.4203C170.975 94.3598 170.572 94.2993 170.11 94.2993C168.9 94.2993 167.747 94.6622 166.709 95.3881C163.942 97.3842 163.134 101.376 165.037 104.34Z" fill="white"/>
+<path d="M220.096 91.6379C223.949 91.6379 227.072 88.3611 227.072 84.319C227.072 80.2768 223.949 77 220.096 77C216.243 77 213.12 80.2768 213.12 84.319C213.12 88.3611 216.243 91.6379 220.096 91.6379Z" fill="white"/>
+<path d="M193.403 95.3881C192.365 94.6622 191.212 94.2993 190.001 94.2993C189.597 94.2993 189.194 94.3598 188.733 94.4203C187.118 94.7832 185.735 95.751 184.812 97.2027L174.896 112.929C173.973 114.381 173.685 116.075 173.973 117.768C174.319 119.462 175.242 120.914 176.625 121.881C178.009 122.849 179.623 123.152 181.238 122.849C182.852 122.486 184.236 121.519 185.158 120.067L195.075 104.34C195.997 102.889 196.285 101.195 195.997 99.5012C195.709 97.8076 194.786 96.3559 193.403 95.3881Z" fill="white"/>
+<path d="M184.062 99.5012C183.716 101.195 184.062 102.889 184.985 104.34L194.901 120.067C195.824 121.519 197.207 122.486 198.821 122.849C200.436 123.212 202.108 122.849 203.434 121.882C206.259 119.885 207.066 115.893 205.163 112.929L195.247 97.2027C194.325 95.751 192.941 94.7832 191.327 94.4203C190.923 94.3598 190.519 94.2993 190.058 94.2993C188.847 94.2993 187.694 94.6622 186.657 95.3881C185.331 96.3559 184.408 97.8076 184.062 99.5012Z" fill="white"/>
+<path d="M204.818 97.2027L194.902 112.929C193.979 114.381 193.691 116.075 193.979 117.768C194.325 119.462 195.248 120.914 196.631 121.881C198.015 122.849 199.629 123.152 201.244 122.849C202.858 122.486 204.242 121.519 205.164 120.067L215.08 104.34C216.925 101.376 216.176 97.3237 213.351 95.3881C212.313 94.6622 211.16 94.2993 209.949 94.2993C209.546 94.2993 209.142 94.3598 208.681 94.4203C207.067 94.7832 205.683 95.751 204.818 97.2027Z" fill="white"/>
+<path d="M204.876 104.34L214.792 120.067C215.715 121.519 217.098 122.486 218.713 122.849C220.327 123.212 221.999 122.849 223.325 121.882C226.15 119.885 226.957 115.893 225.054 112.929L215.138 97.2027C214.216 95.751 212.832 94.7832 211.218 94.4203C210.814 94.3598 210.411 94.2993 209.949 94.2993C208.739 94.2993 207.585 94.6622 206.548 95.3881C203.78 97.3842 203.031 101.376 204.876 104.34Z" fill="white"/>
```

Comment:
looks like the migration worked but there is only one placeholder image, maybe we can use all they provided! should be quick to choose a random image from all the placeholders!

Reply:
Instead of using a random seed, I went with a simpler deciding method that simply uses the index of the card. Randomness in Elm is a bit annoying, and this has a similar effect while being simpler. Hope it's ok :)

Reply:
I think it is! very elegant solution